front-end/src/components/profile/profile.js

```diff
@@ -0,0 +1,11 @@
+export default class Profile {
+    render() {
+        return `
+            <h2>My Profile</h2>
+            <p>Username: MyUsername</p>
```

Comment:
<h3></h3>

I think it would make more sense to use a h3 tag here than p tag?

Reply:
This is just initial. Profile section is delegated to other group members so they can style it how they like.